src/index.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly do not use any `comments` to skip the linter errors, rather fix this in the code. Every linter error has its fix, which may be in the syntax or the format of the code.